armbian/base/scripts/bbb-config.sh

```diff
@@ -125,7 +125,12 @@ case "${COMMAND}" in
 
                 redis_set "bitcoind:listen" "${ENABLE}"
                 generateConfig "bitcoin.conf.template"
-                systemctl restart bitcoind.service
+
+                # only restart bitcoind if system has been configured with setup routine
+                if [[ "$(redis_get 'base:setup')" -eq 1 ]]; then
```

Comment:
Could use `base:bitcoind-services:enabled` instead of `base:setup`.

Reply:
The key `base:bitcoind-services:enabled` should work as well, but I think coupling it to the finished setup is clearer. Not sure if we need `base:bitcoind-services:enabled` at all, at the momement it's only used to re-enable the services on every boot.